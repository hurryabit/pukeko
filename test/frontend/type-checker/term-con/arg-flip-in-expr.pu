data T a b = | C a b
f :: a -> b -> T a b
f x y = C y x
