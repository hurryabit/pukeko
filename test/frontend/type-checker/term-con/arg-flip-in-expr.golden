test/frontend/type-checker/term-con/arg-flip-in-expr.pu:3:1: mismatching types a and b
